ci/sign-all-tasks.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
```

Comment:
We need to make sure this runs when we build a single task by passing --task as well. Should be in `build-single-steps.yml`.

Reply:
I added this to the single steps as well